### PR TITLE
Disable KMS automatic config reload

### PR DESCRIPTION
**Pull request by riendeau**
Don't reload on config changes to prevent issues with upgrade scenarios in the future (removing kms v1 provider). That is, for the rolling upgrade, we don't want the original containers to crash if the kms-config changes are incompatible.

### CODE REVIEW

File: assets/kube-apiserver/kube-apiserver-deployment.yaml [hunk 1]
```diff
@@ -158,6 +158,7 @@ spec:
         - "--openshift-config=/etc/kubernetes/apiserver-config/config.yaml"
 {{- if .KPInfo }}
         - "--encryption-provider-config=/etc/kubernetes/kms-config/config.yaml"
+        - "--encryption-provider-config-automatic-reload=false"
```
Comment (rtheis) [inline assets/kube-apiserver/kube-apiserver-deployment.yaml:161]: You need to run `make bindata` in order to regenerate the bindata based on this change.

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/kube-apiserver/kube-apiserver-deployment.yaml (hunk 1 above)
Comment (riendeau) [reply assets/kube-apiserver/kube-apiserver-deployment.yaml:161]: Updated in https://github.com/openshift/ibm-roks-toolkit/pull/947/commits/2dc7e5fb667f5fea8c4cd8f260db9b8d8e473939

**Pull request merged**